definitions/npm/@apollo/client_v3.x.x/flow_v0.142.x-/client_v3.x.x.js

```diff
@@ -2777,3 +2700,130 @@ declare module "@apollo/client/testing" {
+    static split(predicate: (document: DocumentNode) => boolean, left: DocumentTransform, right?: DocumentTransform): DocumentTransform;
+    constructor(transform: TransformFn, options?: DocumentTransformOptions): this;
+    transformDocument(document: DocumentNode): DocumentNode;
+    concat(otherTransform: DocumentTransform): DocumentTransform;
+    getStableCacheEntry(document: DocumentNode): {|
+      key: DocumentTransformCacheKey;
+      value?: DocumentNode | void;
+    |} | void;
+  }
+
+  // @apollo/client/utilities/graphql/fragments.d.ts
+
+  declare export function getFragmentQueryDocument(document: GraphQL$DocumentNode, fragmentName?: string): GraphQL$DocumentNode;
+  declare export interface FragmentMap {
+    [fragmentName: string]: GraphQL$FragmentDefinitionNode;
+  }
+  declare export type FragmentMapFunction = (fragmentName: string) => GraphQL$FragmentDefinitionNode | null;
+  declare export function createFragmentMap(fragments?: GraphQL$FragmentDefinitionNode[]): FragmentMap;
+  declare export function getFragmentFromSelection(
+    selection: GraphQL$SelectionNode,
+    fragmentMap?: FragmentMap | FragmentMapFunction
+  ): GraphQL$InlineFragmentNode | GraphQL$FragmentDefinitionNode | null;
+
+  // @apollo/client/utilities/graphql/getFromAST.d.ts
+
+  declare type DocumentNode = GraphQL$DocumentNode;
+  declare type OperationDefinitionNode = GraphQL$DocumentNode;
+  declare type FragmentDefinitionNode = GraphQL$DocumentNode;
+
+  declare export function checkDocument(doc: DocumentNode): DocumentNode;
+
+  declare export function getOperationDefinition(doc: DocumentNode): OperationDefinitionNode | void;
+
+  declare export function getOperationName(doc: DocumentNode): string | null;
+
+  declare export function getFragmentDefinitions(doc: DocumentNode): FragmentDefinitionNode[];
+
+  declare export function getQueryDefinition(doc: DocumentNode): OperationDefinitionNode;
+
+  declare export function getFragmentDefinition(doc: DocumentNode): FragmentDefinitionNode;
+
+  declare export function getMainDefinition(queryDoc: DocumentNode): OperationDefinitionNode | FragmentDefinitionNode;
+
+  declare export function getDefaultValues(definition: OperationDefinitionNode | void): { [key: string]: any };
+
+  // @apollo/client/utilities/graphql/storeUtils.d.ts
+
+  declare export interface Reference {
+    +__ref: string;
+  }
+  declare export function makeReference(id: string): Reference;
+  declare export function isReference(obj: any): boolean;
+  declare export type StoreValue = number | string | string[] | Reference | Reference[] | null | void | void | Object;
+  declare export interface StoreObject {
+    __typename?: string;
+    [storeFieldName: string]: StoreValue;
+  }
+  declare export function isGraphQL$DocumentNode(value: any): boolean;
+  declare export function valueToObjectRepresentation(
+    argObj: any,
+    name: GraphQL$NameNode,
+    value: GraphQL$ValueNode,
+    variables?: Object
```

Comment:
Object

Reply:
Fixed in a7631809327d6a1d516410919564b2e7aa94f6f5